### PR TITLE
Add example automated reingest script & docs

### PR DESCRIPTION
**Pull request by Hwesta**
Add a script showing how the Archivematica & storage service API can be called to start and approve reingest. Split the docs into an automated transfers & automated reingest section.

Relates to artefactual/archivematica#391 and artefactual/archivematica-storage-service#100

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,19 @@ Automation Tools
 
 The Automation Tools project is a set of python scripts, that are designed to automate the processing of transfers in an Archivematica pipeline.
 
-Currently, the only automation tool is automate transfers.  It is used to prepare transfers, move them into the pipelines processing location, and take actions when user input is required.  Only one transfer is sent to the pipeline at a time, the scripts wait until the current transfer is resolved (failed, rejected or stored as an AIP) before automatically starting the next available transfer. 
+
+Automated Transfers
+------------------
```
Comment (sevein) [inline README.md]: `###`?

Comment (sevein) [inline README.md]: Are you using dashes on purpose? I thought that was a rst thing but it seems to work.

**Comment by sevein** [comment]
Very clear! 👍

File: README.md (hunk 1 above)
Comment (Hwesta) [inline README.md]: We used dashes elsewhere in the document for h1 and h2, so I continued to use == and -- for the first two levels, and ### for the rest.  Markdown accepts either (https://daringfireball.net/projects/markdown/basics).  It's also a more obvious visual header than just ### prepended.

**Comment by Hwesta** [comment]
This needs to be updated for #21

**Comment by jrwdunham** [comment]
@ross-spencer @sevein This PR is getting old and [recent work from Ross](https://github.com/artefactual/automation-tools/pull/58) may render it obsolete. One of us needs to do some analysis here and doe one of the following I think:

a) justify and document the closing of this branch; or
b) extract any useful (novel) additions from this PR and create a new PR that brings just those in.

**Comment by ross-spencer** [comment]
This is my related comment from a previous code-review which might be relevant for doing some more thorough due-diligence: https://github.com/artefactual/automation-tools/pull/57#discussion_r185214677

**Comment by ross-spencer** [comment]
Finally getting around to this. The current reingest script, documented here: https://github.com/artefactual/automation-tools#reingest focuses entirely on `full-reingest` taking an AIP and running it through Archivematica from the beginning. 

The capability that this PR has, that the new `reingest.py` doesn't is that it supports a `metadata-only` reingest with these lines in-particular highlighting one approach to doing that:

https://github.com/artefactual/automation-tools/blob/31a186887fc8886446604e7791710a2bbd21de7f/reingest/reingest.py#L140-L159

We haven't established a best-practice developed around deleting branches/closing PRs/maintaining novel features in prior-work. @sevein do you have an opinion?

**Comment by sevein** [comment]
> We haven't established a best-practice developed around deleting branches/closing PRs/maintaining novel features in prior-work.

I recently close every abandoned PR in archivematica.git replacing them with issues describing the bug/feature being addressed. These come to mind: https://github.com/artefactual/archivematica/pull/364 and https://github.com/artefactual/archivematica/pull/274.

**Comment by ross-spencer** [comment]
Closing as this work is not being actively worked on. I have captured the feature/requirement expressed here as best as possible in this ticket in the Archivematica issues repository: https://github.com/archivematica/Issues/issues/496

**Pull request closed without merging**